pkg/k8sutil/k8sutil.go

```diff
@@ -203,3 +204,64 @@ func SanitizeVolumeName(name string) string {
 	}
 	return strings.Trim(name, "-")
 }
+
+// TruncateVolumeName truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec)
+func TruncateVolumeName(name string) string {
```

Comment:
This function duplicates most of the functionality from the `SanitizeVolumeName` func above. That func makes a volume name DNS1123 compliant. These funcs should either be reconciled or, we should just use that one wholesale where this new one was used.

Reply:
I agree. I fixed my mistake.